File: 3st/Hyeonseo/programmer/원하는 문자열 찾기.py
def solution(myString, pat):
    # 대소문자 통일 (여기서는 소문자로 변환)
    myString = myString.lower()
    pat = pat.lower()
    
    # 부분 문자열 존재 여부 확인
    if pat in myString:
        return 1
    else:
        return 0

File: 4st/Hani/programmer/원하는 문자열 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        int answer = 0;
        int count = 0;
        
        myString = myString.toLowerCase();
        pat = pat.toLowerCase();
        
        for (int i = 0; i < myString.length() - pat.length(); i++) {
            for (int j = 0; j < pat.length(); j++) {
                if (myString.charAt(i + j) == pat.charAt(j)) {
                    count++;
                }
            } 
        }
        if (count == pat.length()) {
            answer = 1;
        }
    
        return answer;
    }
}